src/xapi-youtube-statements.js

```diff
@@ -57,45 +56,48 @@
           default:
         }
         ADL.XAPIYoutubeStatements.onStateChangeCallback(e, stmt);
-      }
+      };
 
       function playVideo(ISOTime) {
-        var stmt = {"actor":actor, "object": videoActivity};
-        /*if (competency) {
-          stmt["context"] = {"contextActivities":{"other" : [{"id": "compID:" + competency}]}};
-        }*/
-
-        stmt["verb"] = {"id": "http://activitystrea.ms/schema/1.0/play"};
-        stmt["result"] = {"extensions":{"http://id.tincanapi.com/extension/starting-point":ISOTime}};
-        return stmt;
-      }
+            var stmt = {
+                "actor": actor,
+                "object": videoActivity,
+                "verb": { "id": "http://activitystrea.ms/schema/1.0/play", "display": { "en-US": "play" } },
```

Comment:
I think the display should be "played" instead of "play"

"Andrew played some video"